lib/webmock/http_lib_adapters/net_http.rb

```diff
@@ -72,78 +72,57 @@ def constants(inherit=true)
         end
 
         def request(request, body = nil, &block)
+          return super unless started?
```

Comment:
This lets us avoid having to deal with the case where the connection is not started yet.

see https://github.com/ruby/net-http/blob/196f3d70f9ce154dc63a86a2cdbf1aec4df4982e/lib/net/http.rb#L1568-L1574

Reply:
Clever! 👍